homeassistant/components/tessie/sensor.py

```diff
@@ -222,4 +220,4 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the state of the sensor."""
-        return self.entity_description.value_fn(self.value)
+        return self.entity_description.value_fn(super().native_value)
```

Comment:
I generally prefer to avoid super() calls by using another name if the function is never going to be called as the current pattern makes the code harder to follow.

If you wanted to use another name, I'd rename `value` to `_value`

Reply:
Is it okay to have _value and native_value in the class doing the same thing?

Reply:
I'm assuming `_value` would be in the base class and `native_value` would be in the `sensor` subclass?

If so, yes 👍

Reply:
> I'm assuming `_value` would be in the base class and `native_value` would be in the `sensor` subclass?
> 
> If so, yes 👍 

I'd agree, but my intention was to put native_value in the base class so it can be reused by multiple platforms, but maybe this is the wrong approach, and I'll just implement native_value to return _value.

I have to rename value regardless as it conflicts with some other platform base classes.

I thought super was a cleaner solution but I can see how it's more ambiguous.

Reply:
I have used `_value` instead.